packages/apps/plugins/plugin-deck/src/components/DeckLayout.tsx

```diff
@@ -273,7 +273,36 @@ export const DeckLayout = ({
   };
 
   const activeId = Array.from(attention.attended ?? [])[0];
-  const activeNode = activeId ? graph.findNode(activeId) : undefined;
+  const activeNode = useNode(graph, activeId);
+
+  const expandNode = useMemo(
+    () => async (node: Node) => {
+      await graph.expand(node, 'outbound', ACTION_GROUP_TYPE);
+      await graph.expand(node, 'outbound', ACTION_TYPE);
+    },
+    [graph],
+  );
+
+  // TODO(Zan): Maybe this should be a hook?
```

Comment:
I think we should aim to have graph expansion be triggered by user action...but that might not always be possible. In which case, yeah this would probably make sense as a hook.